webpack.config.js

```diff
@@ -118,7 +118,6 @@ const commonConfig = (env) => {
       alias: {
         "react": path.resolve('./node_modules/react'),
         "react-dom": path.resolve('./node_modules/react-dom'),
-        "@patternfly/react-core": path.resolve('./node_modules/@patternfly/react-core')
```

Comment:
@lordrip What is the impact of removing it? (apart from the good news that it allows compilation despite kie using patternfly v4)

Reply:
if I understand well, now if kie-tools-core is using some patternfly component they will use Patternfly v4 for it, right?
(I would need to search in which case they are used and if it can occur in our context to check that it is working)

Reply:
Yes, it will and it's happening already for the components used by the hotkeys panel (which we don't see) and the initial loading distractor